test_conformance/events/test_event_dependencies.cpp

```diff
@@ -97,7 +97,7 @@ int test_event_enqueue_wait_for_events_run_test(
         error = clGetDeviceIDs(platform, CL_DEVICE_TYPE_ALL, 2, two_device_ids,
                                &number_returned);
         test_error(error, "clGetDeviceIDs for CL_DEVICE_TYPE_ALL failed.");
-        if (number_returned != 2)
+        if (number_returned >= 2)
```

Comment:
Is this check backwards?  I think we'd want to skip this test if we get fewer than two devices, not if we get two or more devices.

```suggestion
        if (number_returned < 2)
```

Reply:
Agreed, that is what was intended. Thanks Ben.